app/controllers/emails_controller.rb

```diff
@@ -49,11 +52,12 @@ def send_application_result_email
       @event.set_status_notification_flag_for_applications_with_status(application_letter_status)
```

Comment:
Maybe this should be within the if clause. If you send emails with a status other than acceptance/rejection, you don't want the flag to be updated.